.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+*     @grant @jskeet
```

Comment:
Thanks Justin!
Could we use a GitHub team? Like google cloud events admins — I think it exists.

I can't seem to search for the team name on mobile but again I think that is the name.

Reply:
Sure thing!